DEV-GUIDE.md

```diff
@@ -236,6 +237,38 @@ Push the containers to your registry:
 make push
 ```
 
+### Building and pushing a specific container.
+
+If you want to make a local change and test some component, say istio-ca, you
+could do:
+
+Under istio/istio repo
+
+```shell
+pwd
+```
+The path should be
+
+```shell
+.../istio.io/istio
```

Comment:
.../src/istio.io/istio even but I expect this is covered in the rest already

Reply:
Done